packages/user-ui/src/pages/projects/_id/components/project_header/project_header.rs

```diff
@@ -0,0 +1,130 @@
+use dioxus::prelude::*;
+use dioxus_logger::tracing;
+use dioxus_translate::{translate, Language};
+use models::{deliberation_project::DeliberationProject, DeliberationStatus};
+
+use super::i18n::HeaderTranslate;
+use crate::{
+    components::{
+        icons::{
+            adopted::Adopted, in_progress::InProgress, right_arrow::RightArrow, waiting::Waiting,
+        },
+        label::Label,
+    },
+    utils::time::formatted_timestamp,
+};
+
+use super::super::super::page::Tab;
+
+#[component]
+pub fn ProjectHeader(
+    lang: Language,
+    deliberation: DeliberationProject,
+    active_tab: Signal<Tab>,
+) -> Element {
+    let tr: HeaderTranslate = translate(&lang);
+
+    tracing::debug!("active_tab_value: {:?}", deliberation);
+    let started_at = formatted_timestamp(lang, deliberation.started_at);
+    let ended_at = formatted_timestamp(lang, deliberation.ended_at);
+
+    rsx! {
+        div { class: " max-w-1300 h-fit mb-40 flex max-[1000px]:flex-col-reverse flex-row w-full justify-center items-center gap-40 px-10",
+            // TODO: connect to data and UI
+            //data section
+            div { class: "w-full max-w-720 h-260 flex flex-col justify-center",
+                div { class: "flex flex-col justify-start",
+                    div { class: "w-full flex justify-start items-center font-medium text-lg/24 gap-8 h-fit",
+                        div { class: "w-24 h-24",
+                            match deliberation.status {
+                                DeliberationStatus::Ready => rsx! {
+                                    Waiting {}
+                                },
+                                DeliberationStatus::InProgress => rsx! {
+                                    InProgress {}
+                                },
+                                DeliberationStatus::Finish => rsx! {
+                                    Adopted {}
+                                },
+                            }
+                        }
+
+                        span { "{started_at} ~ {ended_at}" }
+                    }
+                    div { class: "w-full flex justify-start items-center font-semibold text-[32px]/60",
+                        "{deliberation.title}"
```

Comment:
```suggestion
                        {deliberation.title}
```

숫자를 사용하는 경우에는 한계가 있긴할텐데, string 으로 쓰는 경우에는 이렇게 써주세요. 그래야지 lookup 이나 refactoring 이 같이 잘 먹히더라고요.